pepper-apis/src/main/java/org/broadinstitute/ddp/util/DateTimeUtils.java

```diff
@@ -24,4 +27,16 @@ public static Duration parseTimeAmountDuration(String timeAmount) {
         }
         return Duration.parse(timeAmount);
     }
+
+    public static ZoneId parseUserTimeZone(String givenTimeZone) {
```

Comment:
We can rename this to just `parseTimeZone` to be more general.

Reply:
Renamed